scanner/Makefile

```diff
@@ -0,0 +1,170 @@
+# Store tooling in a location that does not affect the system.
+GOBIN := $(CURDIR)/.gobin
+export GOBIN
+PATH := $(GOBIN):"$(PATH)"
+export PATH
+
+# Set to empty string to echo some command lines which are hidden by default.
+SILENT ?= @
+
+SHELL = /bin/bash
+BASE_DIR = $(CURDIR)
+
+TAG := # make sure tag is never injectable as an env var
+
+ifdef CI
+ifneq ($(NIGHTLY_TAG),)
+	TAG := $(NIGHTLY_TAG)
+else ifneq ($(RELEASE_TAG),)
+	TAG := $(RELEASE_TAG)
+endif
+endif
+
+ifeq ($(TAG),)
+	TAG = $(shell git describe --tags --abbrev=10 --dirty --long --exclude '*-nightly-*')
+endif
+
+GO_VERSION := $(shell cat ../EXPECTED_GO_VERSION | cut -c 3-)
+BUILD_IMAGE := docker.io/library/golang:$(shell echo ${GO_VERSION})
```

Comment:
Are we using this?

Should we update the GitHub action to use the golang's image instead of `image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.59`?